sanic/src/feature_extraction.py

```diff
@@ -0,0 +1,49 @@
+import cv2
+import mediapipe as mp
+
+mp_hands = mp.solutions.hands
+
+# Python Generator that yields an image every call
+def getImagesFromVideo(video):
```

Comment:
Hvordan fungere yield?

Reply:
Basically hver gang du kalder funktionen fortsætter den fra yield. (efter første kaldt self)

Reply:
Så skal den vel hedde getImageFromVideo()? 
Men aight

Reply:
Ye i guess det kunne den hedde, men synes i context af et for-loop som er dens primary use-case, så giver det mening med images.. I don't really care tho xd